lib/aggregate/attribute/bitfield.rb

```diff
@@ -40,6 +40,10 @@ def validation_errors(_value)
     []
   end
 
+  def skip_default?
```

Comment:
the name of this method leaves me with lots of questions. It isn't really clear what we are skipping the default for.

Side note - do we always want to skip the default? Or do we only care about skipping the default when the default is actually `nil`?

Reply:
'do_not_write_default_values?' is closest to its current form, but I would invert it.  'write_default_values?' is easier to understand.